svc-events/events/publishevttosubscriber.go

```diff
@@ -147,6 +148,13 @@ func PublishEventsToDestination(data interface{}) bool {
 				go updateSystemPowerState(uuid, inEvent.OriginOfCondition.Oid, inEvent.MessageID)
 				flag = true
 			}
+		} else if strings.EqualFold("ResourceAdded", message.Events[0].EventType) || strings.EqualFold("ResourceRemoved", message.Events[0].EventType) {
```

Comment:
if these two event types added as part to check to rediscovery, IMHO make it consistent across all bmc in default subscription in these two event type subscription need to be added

Reply:
There is no Dell specific checks so this is OK. HPE servers will not trigger this code.